app/services/change_schedule.rb

```diff
@@ -143,11 +143,34 @@ def update_npq_application_cohort!
     participant_profile.npq_application.update!(cohort: new_schedule.cohort)
   end
 
+  def update_funded_place!
+    return unless FeatureFlag.active?(:npq_capping)
+    return unless npq_contract.funding_cap&.positive?
+
+    application = participant_profile.npq_application
+    application.update!(funded_place: application.eligible_for_funding)
+  end
+
+  def npq_contract
```

Comment:
discussed in person: if we can move maybe to the npq lead provider model or a service